papermill/cli.py

```diff
@@ -67,13 +53,9 @@ def papermill(notebook_path, output_path, parameters, parameters_raw,
         for name, value in parameters_raw:
             parameters_final[name] = value
 
-    execute_notebook(
-        notebook_path,
-        output_path,
-        parameters_final,
-        kernel_name=kernel,
-        progress_bar=progress_bar,
-        log_output=log_output)
+    execute_notebook(notebook_path, output_path, parameters_final,
+                     kernel_name=kernel, progress_bar=progress_bar,
+                     log_output=log_output)
```

Comment:
Looks like we need to cover this area, since codecov is reporting this in red (I use the chrome extension). You might already be tackling this in #87 though.

Reply:
Yeah, if someone has a fully functioning correct notebook that can be used for testing that would be helpful.